python/lsst/daf/base/propertyContainer/propertyContainerContinued.py

```diff
@@ -501,6 +534,9 @@ def __setitem__(self, name, value):
             value = ps
         self.set(name, value)
 
+    def __getitem__(self, name):
+        return self.getScalar(name)
+
```

Comment:
Please document what this does in a doc string, if you can figure out a sensible place to put it. I suspect for it to be easily understood it should be in the doc string for the class, but maybe a doc string here would work if sphinx is smart enough.

Reply:
daf_base still uses doxygen and I think the best place to document the dict-like behavior of PropertySet would be in the index.rst file once it's ported to sphinx. There doesn't seem to be a doxygen root page that I can augment.